### PR TITLE
RDKCMF-8861 RDKShell: add support for LISA to launchApplication()

### PR DESCRIPTION
**Pull request by sverkoye**
* Add LISA support: https://github.com/stagingrdkm/LISA
* As alternative to Packager. First LISA is checked. If not supported it falls back to Packager.

### CODE REVIEW

File: RDKShell/RDKShell.cpp [hunk 1]
```diff
@@ -1509,6 +1509,13 @@ namespace WPEFramework {
             return make_shared<WPEFramework::JSONRPC::LinkType<WPEFramework::Core::JSON::IElement>>("Packager.1", "", false, query);
         }
 
+        std::shared_ptr<WPEFramework::JSONRPC::LinkType<WPEFramework::Core::JSON::IElement>> RDKShell::getLISAPlugin()
```
Comment (npoltorapavlo) [inline RDKShell/RDKShell.cpp:1512]: this violates `7. Inter-plugin communication` of the [readme](https://github.com/rdkcentral/rdkservices#readme)

**Review by npoltorapavlo — CHANGES_REQUESTED** [review]

File: RDKShell/RDKShell.cpp (hunk 1 above)
Comment (sverkoye) [reply RDKShell/RDKShell.cpp]: @npoltorapavlo , updated the call to LISA to use JSONRPCDirectLink and thus QueryInterfaceByCallsign()

**Comment by anand-ky** [comment]
@binuinbaraj - Pls review

**Comment by anand-ky** [comment]
Closing older PRs. Pls bring changes to latest sprint.

**Pull request closed without merging**